site/content/docs/manifest/scheduled-job.en.md

```diff
@@ -120,6 +120,21 @@ platform:
   osfamily: windows_server_2019_full
   architecture: x86_64
 ```
+```yaml
+platform:
+  osfamily: windows_server_2019_full
```

Comment:
Should this be `windows_server_2019_core`?

Reply:
Good catch.